pkg/storage/storage.go

```diff
@@ -314,6 +317,9 @@ type ChangelogBackend interface {
 	// if no changes are found, it should return storage.ErrNotFound and an empty continuation token.
 	// It the objectType and the type in the continuation token don't match, it should return ErrMismatchObjectType.
 	ReadChanges(ctx context.Context, store string, filter ReadChangesFilter, options ReadChangesOptions) ([]*openfgav1.TupleChange, []byte, error)
+
+	// CreateContinuationToken creates a continuation token for the given ULID and object type.
+	CreateContinuationToken(ulid string, objType string) ([]byte, error)
```

Comment:
Belongs to the Changelog interface but its still in a shared package, please rename to something like `CreateChangelogContinuationToken` as other backends might start embracing this pattern and names would collide.